dts/riscv/it8xxx2.dtsi

```diff
@@ -276,6 +276,28 @@
 			interrupt-parent = <&intc>;
 		};
 
+		ite_uart1_wrapper: uartwrapper@f02720 {
+			compatible = "ite,it8xxx2-uart";
+			reg = <0x00f02720 0x0020>;
+			status = "disabled";
+			label = "UART1_WRAPPER";
+			port-num = <1>;
+			wakeup-pin = <0>;   /* GPB0 */
+			gpio-dev = <&gpiob>;
```

Comment:
Why isn't this using something like `wakeup-gpios = <&gpiob 0 ...>;`?

Reply:
You are right, we have planned to create another PR to add a WUI node in dtsi, and write the gpio node and pin to be configured like this. It will be more flexible to configure WUI for other drivers.

Reply:
This change has been fixed to `<&gpiob 0 ...>;`